website/docs/r/key_vault.html.markdown

```diff
@@ -83,6 +83,13 @@ The following arguments are supported:
     Azure Resource Manager is permitted to retrieve secrets from the key vault.
     Defaults to false.
 
+* `enabled_for_soft_delete` - (Optional) Boolean flag to specify whether
+    the key vault is enabled for soft delete. Once enabled you can not disable this setting anymore!
+
+* `enabled_for_purge_protection` - (Optional) Boolean flag to specify whether
+    the key vault is enabled for purge protection. Once enabled you can not disable this setting anymore!
```

Comment:
can we replace `Once enabled you can not disable this setting anymore!` with `Changing this forces a new resource to be created.`